src/store/titres.js

```diff
@@ -108,6 +114,16 @@ export const actions = {
     commit('preferencesSet', { section, params })
   },
 
+  preferencesMapIni({ commit }) {
+    const zoomIni = this.state.titres.preferences.carte.zoom
```

Comment:
Le state est accessible en tant que paramètre de la méthode (cf méthodes au dessus), ça permettra d’enlever le `this`.